pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import { showModal } from './Modal';
+
+import './KeyboardHelpDialog.css';
+
+let keyboardHelpOpen = false;
+
+export function showKeyboardHelpDialog() {
+  if (keyboardHelpOpen) {
+    return;
+  }
+
+  keyboardHelpOpen = true;
+  showModal({
+    title:
+      <span>
+        Keyboard Help (press <kbd>F1</kbd> to open;
```

Comment:
See e.g. Inoreader. They don't have a _title_ per se, as they render this in a footer, but still, the approach is the same — to visually separate how the help dialog can be invoked, from the rest of the shortcuts:

![Image](https://cloud.githubusercontent.com/assets/1728158/17785827/226f4090-6537-11e6-9b05-610733dcd9bf.png)